### ISSUE TITLE
Feature request: Super Game Boy support

### ISSUE THREAD

**Issue by nikku4211** [AntonioND/geometrix#1]
I have a Super Game Boy, and I noticed that this game doesn't have any extra colours when plugged into Super Game Boy.

I think it'd be awesome if there at least was some code to set a palette for Super Game Boy mode, even if you'd be limited to only 3 colours.

Who knows, I might even be willing to make an additional SPC version of your game's music for Super Game Boy mode, if you want.

**Comment by AntonioND** [comment]
I thought about SGB, but I didn't really see the benefit to doing that. All I could really do is set 3 colors for the border, and 3 colors for all the blocks because you can't change the colors fast enough. As far as I understand, the SGB commands are relatively slow...

So, in my opinion, it would be a decent amount of effort for not too much gain.

**Comment by nikku4211** [comment]
That makes sense, but it's possible to use just a single colour for each block, though unfortunately, that is less than half of the amount of colours for the blocks in the GBC version.

And if that's still too much effort for a tiny fraction of your audience, I can understand it not being an official feature.

**Comment by AntonioND** [comment]
To be fair. It's more that I have never done SGB stuff, so I'd be doing it blindly. I don't even have my GBs with me, and I don't have a SGB or SNES, so I can't test on hardware. I need to be able to test whatever I do and, while I appreciate it when other people also test on hardware, I want to do it myself to be sure that everything works. I've got stuck in the past for depending on other people's testing my code.

Not many people care about GB homebrew, even fewer people know about this game, even fewer people have bothered to play with it, even fewer have bothered to test it on hardware, and even fewer have a SGB. I wouldn't be surprised if the number of people disappointed by the lack of SGB support is 3 or less.. xD

So yeah, it's not a trivial task, and the number of people that would benefit from it is just too small. Most people would just play in GBC mode after checking out the SGB mode, anyway, it would be prettier. The only reason why I have GB support in the first place is that (if I remember correctly) beware asked me on IRC when I released the game, and I had nothing better to do so I took it as a challenge. Now there are other things I'd rather do.

In any case, thanks for your interest in the game!
